openml/testing.py

```diff
@@ -115,7 +115,7 @@ def tearDown(self) -> None:
         """Tear down the test"""
         os.chdir(self.cwd)
         try:
-            shutil.rmtree(self.workdir, ignore_errors=True)
```

Comment:
I introduced this in another PR to fix this issue but it did not help. Hence, I removed it here again.